src/modules/paint-ui.js

```diff
@@ -1,68 +1,122 @@
 import Todos from './todos.js';
-import {
-  returnHead,
-  returnTodoForm,
-  generateTodos,
-  returnTodo,
-} from './components.js';
+import { returnHead, returnTodoForm, generateTodos } from './components.js';
 
-const displayTodos = () => {
-  document.querySelector('.todo-wrapper').innerHTML = generateTodos();
+/**
+ *
+ * @param {Event} item event from the targeted todo
+ * @returns
+ */
+const updateTodo = (item) => {
```

Comment:
- [ ] @veronica365 great work implementing the interactive list functionality, however, you forgot to add a new file in your modules that handles this functionality only.
![image](https://user-images.githubusercontent.com/61408860/221143772-fff08bf7-e835-4ba7-a0a6-13cb1b4ea7d8.png)

- [ ] [optional] it would be nice if you further break your code into more modules this will make it easier to read, make your code neat and make it easier for future scaling of your app.